recipes/cpython/all/conanfile.py

```diff
@@ -116,8 +120,12 @@ def requirements(self):
         if self.settings.os != "Windows":
             if not is_apple_os(self):
                 self.requires("util-linux-libuuid/2.39.2")
-            # If crypt.h is detected, it is included in the public headers.
-            self.requires("libxcrypt/4.4.36", transitive_headers=True, transitive_libs=True)
+            # In <3.9 and lower patch versions of 3.9/10/11, crypt.h was exposed in Python.h
+            # This was removed in 3.11 and backported: https://github.com/python/cpython/issues/88914
+            # For the sake of this recipe, we only have later patch versions, so this version check
+            # may be slightly inaccurate if a lower patch version is desired.
+            transitive_crypt = Version(self.version) < "3.9"
```

Comment:
I can, it would have just been a little complex. I also figured it wasn't the end of the world if they ended up being transitive if they didn't have to be.

Reply:
Personally, I don't see much value in adding it now that I think about it. If these were recent patches, I would do so, but xcrypt has been non-transitive for a few years at this point (late 2021), so it would be a very specific requirement on a specific old patch version that would warrant this necessary.